index.html

```diff
@@ -144,97 +155,101 @@ <h2 class="projecttitle">Profesional Art Printing Data</h2>
-                        <li class="skillitem3">Terminal</li>
-                    </ul>
-                    </div>
 
+            <div class="skilllistcontainer">
+                <ul class="skillitem2">
+                    <li class="skillitem2">bootstrap</li>
+                    <li class="skillitem2">Ruby on Rails</li>
+                    <li class="skillitem2">RSPec</li>
+                    <li class="skillitem2">CapyBara</li>
+                    <li class="skillitem2">Selenium</li>
+                </ul>
             </div>
 
-        </section>
-    </section>
+        
+        </div>
+        <div class="skills">
+            <div class="skillstitle">
+                <img src="./img/circle.png" alt="square" class="skills-img">
+                <h2 class="skillstitle">Skills</h2>
+            </div>
+            <div class="skilllistcontainer">
+                <ul class="skillitem3">
+                    <li class="skillitem3">Codekit</li>
+                    <li class="skillitem3">Github</li>
+                    <li class="skillitem3">Codepen</li>
+                    <li class="skillitem3">Gitlab</li>
+                    <li class="skillitem3">Terminal</li>
+                </ul>
+                </div>
 
-</div>
+        </div>
 
-<div><section class="contactme">
-    <h3 id="formtitle">I'm always interested in hearing about new projects, so if you'd like to chat please get in</h3>
-    <form action="https://formspree.io/f/xeqnbagq" method="POST">
-        <input name="full_name" class="contactinput" id="inputname" type="text" maxlength="30" placeholder="Full Name" required>
-        <input name="email" class="contactinput" id="inputemail" type="email" placeholder="Email" required>
-        <textarea name="message" cols="30" rows="10" placeholder="Enter text here." maxlength="500" required></textarea>
-        <input id="getintouch" type="submit" value="Get in touch">
+    </section>
 
-    </form>
+    <section class="contactme">
 
-    <ul id="footericons" class="icons">
-        <li><a href="#"><img src="./icons/Github.svg" alt="github"></a></li>
+        <div class="containercontact">
+            <h3 id="formtitle">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+            <form id="contactform" action="https://formspree.io/f/xeqnbagq" method="POST">
+                <input name="full_name" class="contactinput" id="inputname" type="text" maxlength="30" placeholder="Full Name" required>
+                <input name="email" class="contactinput" id="inputemail" type="email" placeholder="Email" required>
+                <textarea name="message" cols="30" rows="10" placeholder="Enter text here." maxlength="500" required></textarea>
+                <input id="getintouch" type="submit" value="Get in touch">
 
-        <li><a href="#"><img src="./icons/LinkedIn.svg" alt="linkedin"></a></li>
+            </form>
```

Comment:
Great work @dgonzalesi 🤝

- [ ] You may please try to make your design a replica of the design provided in the Figma template. You may align your social icons properly, Reduce the font size of your text accordingly, and also add the dark line that is just above the social icons.
- Your design
![your form](https://user-images.githubusercontent.com/63917725/178077221-8eac3da2-0aad-4db0-94aa-db13fd20ecb2.PNG)

- Figma design
![figma form](https://user-images.githubusercontent.com/63917725/178077332-0ac08200-5920-49fe-ba3a-fd93edd618a7.PNG)